pkg/kubelet/cm/container_manager_unsupported.go

```diff
@@ -31,7 +31,7 @@ type unsupportedContainerManager struct {
 
 var _ ContainerManager = &unsupportedContainerManager{}
```

Comment:
I wonder if we can use the `Stub` implementation instead of having to maintain an `unsupported` version